delegation-toolkit/how-to/create-smart-account.md

```diff
@@ -1,47 +1,33 @@
 ---
-sidebar_label: Configure accounts and signers
-description: Learn how to configure different types of MetaMask Smart Accounts and signers using Viem.
-sidebar_position: 1
+description: Learn how to create a MetaMask smart account using Viem.
+sidebar_position: 2
 ---
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-# Configure MetaMask Smart Accounts and signers
+# Create a MetaMask smart account
 
-The MetaMask Delegation Toolkit supports different [MetaMask smart account types](../../concepts/smart-accounts.md#smart-account-implementation-types),
+The MetaMask Delegation Toolkit is embedded, meaning that the end user can instantly interact with a dapp without wallet authorization, confirmations, or corporate logos.
```

Comment:
This is not true, and doesn't fit right. The toolkit is not embedded, the accounts are embedded. Also, embedded wallet doesn't mean that a dApp doesn't need wallet authorization. It totally depends on the dApp. Since we are a smart account layer, not a signer layer this is not the stand out feature of smart account. 

For instance, if someone is using Privy as a signer for MetaMask smart account, by default it'll show confirmation screens of privy for signing cause Privy by default uses iFrame for signature.